<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCockpitComTransmissionComponent</class>
 <widget class="QFrame" name="CCockpitComTransmissionComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cockpit COM transmission</string>
  </property>
  <layout class="QHBoxLayout" name="hl_CockpitComTransmission">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_Com1">
     <property name="title">
      <string>COM1</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cb_Com1Tx">
        <property name="text">
         <string>transmit</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_Com1Rec">
        <property name="text">
         <string>receive</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_Com1VolumeTx">
        <property name="suffix">
         <string> transmit</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>95</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_Com1VolumeRec">
        <property name="suffix">
         <string> receive</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>95</number>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pb_Com1Save">
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_Com2">
     <property name="title">
      <string>COM2</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cb_Com2Tx">
        <property name="text">
         <string>transmit</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_Com2Rec">
        <property name="text">
         <string>receive</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_Com2VolumeTx">
        <property name="suffix">
         <string> transmit</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>95</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_Com2VolumeRec">
        <property name="suffix">
         <string> receive</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>95</number>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pb_Com2Save">
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
